keras/backend/mxnet_backend.py

```diff
@@ -5254,11 +5254,14 @@ def _get_mxnet_context(context):
             for gpu_id in range(0, context):
                 mxnet_context.append(mx.gpu(gpu_id))
     elif isinstance(context, str):
-        # If user provides GPU context in the format - "gpu(0)" or "eia" i.e., string.
-        if context.lower() == "eia":
-            mxnet_context.append("eia")
+        # If user provides GPU context in the format - "gpu(0)" or "eia" or "eia(0)" i.e., string.
+        if context.lower().startswith('eia('):
+            index = int(context[4:-1])
+            mxnet_context.append(mx.eia(index))
+        elif context.lower() == 'eia':
+            mxnet_context.append(mx.eia())
         else:
-            mxnet_context.append(context)
+            mxnet_context.append(context.lower())
```

Comment:
Isnt this the same:
mxnet_context.append(mx.eia())
mxnet_context.append(context.lower())

Can we re-use the same code for "eia" as "gpu"?

Reply:
Addressed the comments.